packages/react-charting/src/components/GroupedVerticalBarChart/GroupedVerticalBarChartRTL.test.tsx

```diff
@@ -1,13 +1,551 @@
+        color: DefaultPalette.blueLight,
+        legend: 'MetaData3',
+      },
+    ],
+  },
+  {
+    name: '2020',
+    series: [
+      {
+        key: 'series1',
+        data: 54,
+        xAxisCalloutData: '2020/04/30',
+        color: DefaultPalette.accent,
+        legend: 'MetaData1',
+      },
+      {
+        key: 'series2',
+        data: 72,
+        xAxisCalloutData: '2020/04/30',
+        color: DefaultPalette.blueMid,
+        legend: 'MetaData2',
+      },
+      {
+        key: 'series3',
+        data: 18,
+        xAxisCalloutData: '2020/04/30',
+        color: DefaultPalette.blueLight,
+        legend: 'MetaData3',
+      },
+    ],
+  },
+];
+
+const chartPoints = [
+  {
+    name: 'Metadata info multi lines text Completed',
+    series: [
+      {
+        key: 'series1',
+        data: 33000,
+        color: DefaultPalette.blueLight,
+        legend: 'MetaData1',
+        xAxisCalloutData: '2020/04/30',
+        yAxisCalloutData: '33%',
+      },
+      {
+        key: 'series2',
+        data: 44000,
+        color: DefaultPalette.blue,
+        legend: 'MetaData4',
+        xAxisCalloutData: '2020/04/30',
+        yAxisCalloutData: '44%',
+      },
+    ],
+  },
+  {
+    name: 'Meta Data2',
+    series: [
+      {
+        key: 'series1',
+        data: 33000,
+        color: DefaultPalette.blueLight,
+        legend: 'MetaData1',
+        xAxisCalloutData: '2020/05/30',
+        yAxisCalloutData: '33%',
+      },
+      {
+        key: 'series2',
+        data: 3000,
+        color: DefaultPalette.blue,
+        legend: 'MetaData4',
+        xAxisCalloutData: '2020/05/30',
+        yAxisCalloutData: '3%',
+      },
+    ],
+  },
+
+  {
+    name: 'Single line text ',
+    series: [
+      {
+        key: 'series1',
+        data: 14000,
+        color: DefaultPalette.blueLight,
+        legend: 'MetaData1',
+        xAxisCalloutData: '2020/06/30',
+        yAxisCalloutData: '14%',
+      },
+      {
+        key: 'series2',
+        data: 50000,
+        color: DefaultPalette.blue,
+        legend: 'MetaData4',
+        xAxisCalloutData: '2020/06/30',
+        yAxisCalloutData: '50%',
+      },
+    ],
+  },
+];
+
+describe('Grouped Vertical bar chart rendering', () => {
+  testWithoutWait(
+    'Should render the grouped vertical bar chart with numeric x-axis data',
+    GroupedVerticalBarChart,
+    { data: chartDataWithNumaricXAxis },
+    container => {
+      // Assert
+      expect(container).toMatchSnapshot();
+    },
+  );
+
+  testWithoutWait(
+    'Should render the grouped vertical bar chart with string x-axis data',
+    GroupedVerticalBarChart,
+    { data: chartPoints },
+    container => {
+      // Assert
+      expect(container).toMatchSnapshot();
+    },
+  );
+
+  testWithWait(
+    'Should render the vertical bar chart with string x-axis data with given container width and bar width',
+    GroupedVerticalBarChart,
+    { data: chartPoints, width: 1000, barWidth: 1 },
```

Comment:
Does this work. The minimum possible bar width is 16px for GVBC.

Reply:
As i mentioned barwidth is not working as expected for GVBC so that might causes the issue but this test case running without any error.

Reply:
Mention the bug id in the code as a comment